### PR TITLE
Print property loaded from common.properties

### PR DESCRIPTION
**Pull request by ruanwenjun**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Review by SbloodyS — DISMISSED** [review]
LGTM

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java [hunk 1]
```diff
@@ -48,7 +51,12 @@ private PropertyUtils() {
     public static synchronized void loadPropertyFile(String... propertyFiles) {
         for (String fileName : propertyFiles) {
             try (InputStream fis = PropertyUtils.class.getResourceAsStream(fileName);) {
-                properties.load(fis);
+                Properties subProperties = new Properties();
+                subProperties.load(fis);
+                subProperties.forEach((k, v) -> {
+                    logger.info("Get property from {}: {} -> {}", fileName, k, v);
```
Comment (caishunfeng) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java]:
```suggestion
                    logger.info("Get property {} -> {}",  k, v);
```

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java [hunk 2]
```diff
@@ -48,7 +51,12 @@ private PropertyUtils() {
     public static synchronized void loadPropertyFile(String... propertyFiles) {
         for (String fileName : propertyFiles) {
             try (InputStream fis = PropertyUtils.class.getResourceAsStream(fileName);) {
-                properties.load(fis);
+                Properties subProperties = new Properties();
+                subProperties.load(fis);
+                subProperties.forEach((k, v) -> {
+                    logger.info("Get property {} -> {}",  k, v);
```
Comment (kezhenxu94) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java]: Note that there might be sensitive configs such as access key / access id in the `common.properties` file

Comment (ruanwenjun) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java]: Yes, do we have any way to mask this, can we create a mask key list, if the config key in the list, then we didn't print the value?

Comment (kezhenxu94) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java]: Yes but we have to maintain a list of credential keys. I'd like to know why we want to print these properties?

Comment (ruanwenjun) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java]: Sometimes, I suspect that we didn't load the correct configuration file, so I want to print the config here :)

Comment (kezhenxu94) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java]: > Sometimes, I suspect that we didn't load the correct configuration file, so I want to print the config here :)

1. If you can add a break point and evaluate the property in doubt, do that
2. Or use debug level so these won’t be printed by default unless users enable explicitly, then users should be aware of the risk of printing so many detailed logs.

Comment (ruanwenjun) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java]: Thanks for your suggestion, I changed the log level to debug.

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**